docs/apache-airflow/faq.rst

```diff
@@ -441,6 +441,17 @@ If the tasks are not related by dependency, you will need to :ref:`build a custo
 Airflow UI
 ^^^^^^^^^^
 
+Why did my task fail with no logs in the UI?
+--------------------------------------------
+
+Logs are [typically served when a task reaches a terminal state](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/logging-tasks.html#serving-logs-from-workers). Sometimes, a task's normal lifecycle is disrupted, and the task's
+worker is unable to write the task's logs. This typically happens for one of two reasons:
+
+1. [Zombie Tasks](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks)
+2. Tasks failed after getting stuck in queued (Airflow 2.6.0+)
```

Comment:
What the Airflow 2.6.0+ means?
if task is stuck in queue it means it didn't reach worker. "extracting" it from the queue can be done with any Airflow version.

Reply:
#30375 will fail tasks that get stuck in queued for some reason. This will result in missing task logs

Reply:
So we should refer to the relevant Airflow setting task_queued_timeout and ask user to check if the threshold chosen is OK for them